packages/actions/src/helpers/security.ts

```diff
@@ -0,0 +1,38 @@
+import fetch from "node-fetch"
```

Comment:
I think the whole thing should be moved to the `phase2cli` package as the package `actions` should have nothing to do with Github directly. In fact, if you notice in the `actions` package even the functions for handshaking with Firebase use generic OAuth credentials that are a superset of Github's.